post-scan-actions/azure-python-slack-notification/template.json

```diff
@@ -0,0 +1,233 @@
+            }
+        }
+    },
+    "variables": {
+        "location": "[resourceGroup().location]",
+        "functionAppName": "[parameters('fssSlackFunctionName')]",
+        "hostingPlanName": "[parameters('fssSlackFunctionName')]",
+        "applicationInsightsName": "[parameters('fssSlackFunctionName')]",
+        "storageAccountName": "[concat('fssslack', uniquestring(resourceGroup().id))]",
+        "scanResultTopicResourceIDSplits": "[split(parameters('scanResultTopicResourceID'), '/')]",
+        "scanResultTopicResourceSubscriptionID": "[variables('scanResultTopicResourceIDSplits')[2]]",
+        "scanResultTopicResourceGroupName": "[variables('scanResultTopicResourceIDSplits')[4]]",
+        "scanResultTopicNamespace": "[variables('scanResultTopicResourceIDSplits')[8]]",
+        "scanResultTopicName": "[variables('scanResultTopicResourceIDSplits')[10]]",
+        "scanResultTopicNamespaceResourceID": "[split(parameters('scanResultTopicResourceID'), '/topics/')[0]]",
+        "scanResultSASKeyName": "RootManageSharedAccessKey",
+        "scanResultAuthRuleResourceID": "[format('{0}/authorizationRules/{1}', variables('scanResultTopicNamespaceResourceID'), variables('scanResultSASKeyName'))]",
+        "scanResultTopicSubscriptionName": "fss-slack-subscription",
+        "scanResultTopicAPIVersion": "2017-04-01",
+        "functionAppAPIVersion": "2020-12-01",
+        "fssSlackRoleAssignmentDeployment": "fssSlackRoleAssignment",
+        "slackWebHook": "[parameters('webhookURL')]",
+        "slackChannel": "[parameters('slackChannelName')]",
+        "subscriptionName": "[parameters('subscriptionName')]",
+        "scanResultSubscriptionDeployment": "scanResultSubscription"
+    },
+    "resources": [{
+            "type": "Microsoft.Storage/storageAccounts",
+            "apiVersion": "2019-06-01",
+            "name": "[variables('storageAccountName')]",
+            "location": "[variables('location')]",
+            "sku": {
+                "name": "Standard_LRS"
+            },
+            "kind": "Storage"
+        },
+        {
+            "type": "Microsoft.Web/serverfarms",
+            "apiVersion": "2020-12-01",
+            "name": "[variables('hostingPlanName')]",
+            "location": "[variables('location')]",
+            "kind": "linux",
+            "sku": {
+                "name": "Y1",
+                "tier": "Dynamic",
+                "size": "Y1",
+                "family": "Y",
+                "capacity": 1
+            },
+            "properties": {
+                "reserved": true
+            }
+        },
+        {
+            "type": "Microsoft.Web/sites",
+            "apiVersion": "[variables('functionAppAPIVersion')]",
+            "name": "[variables('functionAppName')]",
+            "location": "[variables('location')]",
+            "kind": "functionapp",
+            "identity": {
+                "type": "SystemAssigned"
+            },
```

Comment:
The function doesn't need system assigned identity like quarantine function, which needs the identity to promote/quarantine storage accounts.
```suggestion
```

Reply:
removed